radixdlt-core/radixdlt/src/main/java/com/radixdlt/sync/RemoteSyncService.java

```diff
@@ -133,6 +155,31 @@ private void processLedgerUpdate(LedgerUpdate ledgerUpdate) {
 		final VerifiedLedgerHeaderAndProof updatedHeader = ledgerUpdate.getTail();
 		if (accComparator.compare(updatedHeader.getAccumulatorState(), this.currentHeader.getAccumulatorState()) > 0) {
 			this.currentHeader = updatedHeader;
+			ledgerUpdate.getNextValidatorSet().ifPresent(newValidatorSet -> this.currentValidatorSet = newValidatorSet);
+			this.sendStatusUpdateToSomePeers(updatedHeader);
+		}
+	}
+
+	private void sendStatusUpdateToSomePeers(VerifiedLedgerHeaderAndProof header) {
+		if (!(this.localSyncService.getSyncState() instanceof SyncState.IdleState)) {
```

Comment:
Do we need this check?

Reply:
this is not strictly needed, but I think it doesn't make much sense to broadcast the status message if the node is still syncing itself